src/main/java/com/academia/library/mapper/UserMapper.java

```diff
@@ -0,0 +1,14 @@
+package com.academia.library.mapper;
+
+import com.academia.library.dto.UserRequestDto;
+import com.academia.library.dto.UserResponseDto;
+import com.academia.library.model.User;
+import org.mapstruct.Mapper;
+
+@Mapper(componentModel = "spring")
+public interface UserMapper {
+
+    UserResponseDto toDto(User user);
```

Comment:
Please rename UserResponseDto to UserResponse
Do that for ALL similar names

Reply:
Ok, thank